question16/main.js

```diff
@@ -41,35 +41,34 @@ async function init() {
 }
 init();
 
-function renderNewsUiElement(data) {
-  tabsGroup.appendChild(createTabs(data));
-  newsWrapper.appendChild(createContents(data));
+function renderNewsUiElement(newsData) {
+  tabsGroup.appendChild(getTabListsFragment(newsData));
+  newsWrapper.appendChild(renderContents(newsData));
 }
 
-//Render Tab Element
-const createTabs = (data) => {
+//get Tab Elements 
+const getTabListsFragment = (newsData) => {
   const fragmentTablists = document.createDocumentFragment();
-  for (let i = 0; i < data.length; i++) {
+  for (let i = 0; i < newsData.length; i++) {
     const tabList = createElementWithClassName("li", "tabList");
-    tabList.textContent = data[i].category;
-    i === 0 && tabList.classList.add("is-active-tab");
+    tabList.textContent = newsData[i].category;
+    newsData[i].initialDisplay === true && tabList.classList.add("is-active-tab");
     fragmentTablists.appendChild(tabList);
   }
   return fragmentTablists;
 };
 
 //Render Content Element
-const createContents = (data) => {
+const renderContents = (newsData) => {
   const fragmentContents = document.createDocumentFragment();
-  for (let i = 0; i < data.length; i++) {
-    // const {articles} = data[i];
+  for (let i = 0; i < newsData.length; i++) {
     const contentsContainer = createElementWithClassName("div", "contentsContainer");
     const tabContents = createElementWithClassName("div", "tabContents");
     const tabContentsUl = createElementWithClassName("ul", "tabContentsUl");
-    i === 0 && contentsContainer.classList.add("is-active-content");
+    newsData[i].initialDisplay === true && contentsContainer.classList.add("is-active-content");
```

Comment:
```suggestion
    newsData[i].initialDisplay && contentsContainer.classList.add("is-active-content");
```

Reply:
I have corrected the same as above as you suggested!